internal/namespaces/init/custom_init_ssh_test.go

```diff
@@ -14,10 +14,14 @@ import (
 )
 
 func setUpSSHKeyLocally(key string) core.BeforeFunc {
```

Comment:
IMHO we should remove this function that is only called 2 times and only keep `setUpSSHKeyLocallyWithKeyName`

Reply:
:+1: